Sources/MixedReality/HoloLensCapture/HoloLensCaptureApp/Package.appxmanifest

```diff
@@ -10,7 +10,7 @@
   <Identity
     Name="0e0b1c4d-d6a2-4761-8a1e-8fb7b914a071"
     Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, C=US"
-    Version="1.0.0.0" />
+    Version="1.0.2.0" />
```

Comment:
Just curious, why did this have to change?

Reply:
This auto-increments when new packages are published. Not necessary. I'll revert this.